modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,63 @@
 components:
     schemas:
+        Collaborator:
+            description:
+                Represents a collaborator for an entry.
+            properties:
+                actionIds:
+                    description:
+                        The collaborator actions for the shared asset.
+                    items:
+                        type: string
+                    type: array
+                actions:
+                    additionalProperties:
+                        additionalProperties:
+                            type: string
+                        type: object
+                    # @review
+                    description:
+                        Block of actions allowed by the user making the request.
+                    readOnly: true
+                    type: object
+                creator:
+                    $ref: "../../headless-delivery/headless-delivery-impl/rest-openapi.yaml#Creator"
+                    description:
+                        The object entry folder's creator.
+                    readOnly: true
+                dateExpired:
+                    description:
+                        The expiration date to be a collaborator of the asset.
+                    format: date-time
+                    type: string
+                externalReferenceCode:
+                    # @review
+                    description:
+                        The collaborator external reference code.
+                    type: string
+                name:
+                    description:
+                        The collaborator name.
+                    readOnly: true
+                    type: string
+                portrait:
+                    description:
+                        The collaborator portrait.
+                    readOnly: true
+                    type: string
+                share:
+                    description:
+                        If the collaborator can share or not the asset.
+                    type: boolean
+                type:
```

Comment:
This could be an enum @Mikellorza

Reply:
I will try and I will come back here with the results

Reply:
I am afraid there is no enum field  type for the yaml :( . The validation throws an error like the following: 'Schema validation: Value should be one of: "array", "boolean", "integer", "null", "number", "object", "string" '

Reply:
As we talked in slack, could be like this:

```
viewableBy:
                    description:
                        A write-only property that specifies the document’s default permissions.
                    enum: [Anyone, Members, Owner]
                    type: string
```

Reply:
done mozo, thanks